src/Microsoft.Deployment.DotNet.Releases/tests/ProductCollectionTests.cs

```diff
@@ -28,24 +29,24 @@ public async Task ItThrowsIfPathIsNull()
         {
             Func<Task> f = async () => await ProductCollection.GetFromFileAsync((string)null, false);
 
-            var exception = await Assert.ThrowsAsync<ArgumentNullException>(f);
+            ArgumentNullException exception = await Assert.ThrowsAsync<ArgumentNullException>(f);
         }
 
         [Fact]
         public async Task ItThrowsIfPathIsEmpty()
         {
             Func<Task> f = async () => await ProductCollection.GetFromFileAsync("", false);
 
-            var exception = await Assert.ThrowsAsync<ArgumentException>(f);
-            Assert.Equal("Value cannot be empty (path).", exception.Message);
+            ArgumentException exception = await Assert.ThrowsAsync<ArgumentException>(f);
+            Assert.Equal("Value cannot be empty.\r\nParameter name: path", exception.Message);
```

Comment:
Oh, now, I have not. I believe the repo only builds Windows legs, but I'll update it.

Reply:
These are fixed. I wonder if we need to have Linux test legs enabled on the pipeline at some point

Reply:
I think so since the lib is intended to be x-plat.  I thought the ClickOnce tool was also intended to be xplat except for the signing verb.  @NikolaMilosavljevic - thoughts?

Reply:
Yes, we should enable Linux leg, for testing if not for Linux-specific binaries/packages.

Reply:
@NikolaMilosavljevic - Can you please log an issue to track this work?

Reply:
> @NikolaMilosavljevic - Can you please log an issue to track this work?

https://github.com/dotnet/deployment-tools/issues/91